roles/filebeat/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+filebeat_image: tpolekhin/filebeat
```

Comment:
is this a custom image? if so, is there a github repo and can it be pushed to the ciscocloud organization instead?